### PR TITLE
Add private locations documentation

### PR DESCRIPTION
**Pull request by brampling**

### CODE REVIEW

**Comment by umutuzgur** [comment]
I prefer "Blair's lair" as a private location name

File: site/content/docs/accounts-and-users/adding-private-locations.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: Adding private locations
+weight: 51
+menu:
+  docs:
+    parent: "Accounts & Users"
```
Comment (umutuzgur) [inline site/content/docs/accounts-and-users/adding-private-locations.md:6]: I would actually create a new section for this

File: site/content/docs/accounts-and-users/adding-private-locations.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+7) You now see your new private location in the list with no agents installed. Copy the docker run command as shown.
+
+![private location added](/docs/images/teams/private_location_key.png)
+
+8) Paste the `docker run` command into your container host. Paste your API key from step 6 between the quotation marks for the `API_KEY` environment variable. Optionally, replace `docker` with the command for your container engine of choice (podman, etc.).
+
+9) Optional: You can configure an HTTP proxy if one is required for your environment. Add an additional environment variable after the `API_KEY` like this:
+
+`-e HTTP_PROXY="https://user:password@127.0.0.1:8080"`
+
```
Comment (umutuzgur) [inline site/content/docs/accounts-and-users/adding-private-locations.md:61]: Should we create a "supported env vars" section? We have one more env that we allow users to set JOB_CONCURRENCY and we will probably have more in the future

Comment (brampling) [reply site/content/docs/accounts-and-users/adding-private-locations.md:61]: Right now I just want to get the walkthrough out, I would add this in the next iteration. Same as splitting this out into its own section. wdyt?

File: site/content/docs/accounts-and-users/adding-private-locations.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+Here are the requirements/questions before you get started:
+
+- Container runtime (we test using Docker, but other OCI-compliant runtimes should work)
+- Outbound internet access to https://agent.checklyhq.com (proxy configuration is supported)
+- Access to your internal API- or browser-based application
+- A Checkly account that you will use for testing Private Locations
+
+Here's how private locations work:
+
+A private location is similar to our existing public locations, but you have control over it. A private location can be in your own segregated network,  on-premises or cloud-based. You need one or more Checkly Agents installed in that location/network with access to your applications. Checkly checks are configured and scheduled in the Checkly cloud-based web UI as usual, but selecting a private location runs the check on the agent(s) in your infrastructure. Agents are stateless and ephemeral. As long as you have at least one operational agent, checks will run in your private location. Adding more agents will distribute the load and improve resilience automatically.
```
Comment (umutuzgur) [inline site/content/docs/accounts-and-users/adding-private-locations.md:30]: Should we mention that there are currently differences in terms of chrome between public cloud and private locations?

Comment (umutuzgur) [reply site/content/docs/accounts-and-users/adding-private-locations.md:30]: Let me make this comment irrelevant then. I created a PR so they use the same chromium binaries

**Review by umutuzgur — APPROVED** [review]
LGTM

**Pull request merged**